source/examples/verovio/mspace.mei

```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://music-encoding.org/schema/dev/mei-CMN.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://music-encoding.org/schema/dev/mei-CMN.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="5.1-dev">
+  <meiHead>
+    <fileDesc>
+      <titleStmt>
+        <title>Measure space</title>
+        <editor>
+          <persName>Klaus Rettinghaus</persName>
+        </editor>
+      </titleStmt>
+      <pubStmt>
+        <date isodate="2023">2023</date>
+        <availability>
+          <useRestrict label="licence" auth="https://spdx.org/licenses/" codedval="ECL-2.0">
+            <p>Educational Community License v2.0</p>
+          </useRestrict>
+        </availability>
+      </pubStmt>
+      <seriesStmt>
+        <title>MEI guidelines examples</title>
+      </seriesStmt>
+    </fileDesc>
+  </meiHead>
+  <music>
+    <body>
+      <mdiv n="1">
+        <score>
+          <scoreDef system.leftline="false">
+            <staffGrp>
+              <staffDef n="1" lines="5" clef.shape="G" clef.line="2" meter.sym="common" meter.count="4" meter.unit="4" keysig="4s" />
+            </staffGrp>
+          </scoreDef>
+          <section>
+            <pb />
+            <measure>
```

Comment:
Would be good to have n attribute on all measures here, to see it directly in the encoding snippet.